### ISSUE TITLE
Memory access modifiers

### ISSUE THREAD

**Issue by RTKarbon** [RTKarbon/ele_575_final_project#1]
Do you familiar with these modifiers?

Finally, one can also select whether each
individual global load instruction’s memory access should go
through the L1 cache or not, by using the global memory
access modiﬁers available in PTX 2.x.
------ W. Jia, K. A. Shaw, and M. Martonosi, “Characterizing and improving the use of
demand-fetched caches in gpus,” in Proceedings of the 26th ACM international
conference on Supercomputing. ACM, 2012, pp. 15–24 @felker

**Comment by felker** [comment]
No, I am not. Why would we want to bypass the L1? We could just turn the L1 size to zero?
